gremlin-python/src/main/python/gremlin_python/driver/connection.py

```diff
@@ -61,6 +61,7 @@ def write(self, request_message):
         request_id = str(uuid.uuid4())
         if request_message.args.get("requestId"):
             request_id = request_message.args.get("requestId")
+            uuid.UUID(request_id) # The server will return an error if non-UUID sent.
```

Comment:
```suggestion
            uuid.UUID(request_id) # checks UUID is not malformed as the server will return an error if non-UUID sent.
```
Thought I'd add a bit more clarification that this is doing a check and not the transformation. I was initially wondering why you didn't assign it back to request id.